metadata.json

```diff
@@ -4,20 +4,58 @@
   "author": "Dominik Richter",
   "summary": "Configures the base OS with hardening",
   "license": "Apache-2.0",
-  "source": "https://github.com/TelekomLabs/puppet-os-hardening",
-  "project_page": "https://github.com/TelekomLabs/puppet-os-hardening",
-  "issues_url": "https://github.com/TelekomLabs/puppet-os-hardening/issues",
-  "description": "Configures the base OS with hardening",
-  "types": [
-
+  "source": "https://github.com/dev-sec/puppet-os-hardening.git",
+  "project_page": "https://github.com/dev-sec/puppet-os-hardening",
+  "issues_url": "https://github.com/dev-sec/puppet-os-hardening/issues",
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 3.0.0 < 5.0.0"
```

Comment:
I just checked my agent versions - with the last upgrade on Puppet Enterprise I partly rolled out puppet agent v5, so I think I could raise this to < 6.0.0